### ISSUE TITLE
Not getting good results after training on own dataset

### ISSUE THREAD

**Issue by abdur4373** [dontLoveBugs/FCRN_pytorch#5]
Hello @dontLoveBugs  Actually I have prepared my own data set of indoor scene in my environment and want to train model on that. I am freezing all other layers except for the up projection blocks and the result is not so good. Even I trained it on as small data set as 600 images and achieved 82 percent accuracy but the results were not good visually.
I donot know the reason of that maybe you can suggest me something.
And the images I want to train are approximately 6k. 
The pretrained weights with NYU are even performing better.
    batch_size = 32
    learning_rate = 1.0e-3
    monentum = 0.9
    weight_decay = 0.0005
    num_epochs = 70
 optimizer = torch.optim.SGD(filter(lambda p: p.requires_grad, model.parameters()), lr=learning_rate,
                                momentum=monentum, weight_decay=weight_decay)
and lr is halved after 10 epochs.

![MS_LAB_269_unfilled](https://user-images.githubusercontent.com/46248215/60284606-5fb44480-9925-11e9-9032-a6e4f90f9062.png)

Validation depth image
![Screenshot from 2019-06-27 21-37-28](https://user-images.githubusercontent.com/46248215/60284080-19121a80-9924-11e9-9c73-2689e3ed9831.png)

rgb image
![Screenshot from 2019-06-27 21-44-59](https://user-images.githubusercontent.com/46248215/60284489-1e239980-9925-11e9-9481-e6e043afafb7.png)

**Comment by dontLoveBugs** [comment]
Maybe your dataset is too small. If you want to train your model on a new indoor scene dataset, I think to finetune the pretrained model of NYU dataset in your indoor scene dataset is a feasible approach. I don't know your "accuracy" means what,  "rml", "rmse" or "pixel accuracy"? Besides, the depth range of your test image seems not large, which may cause the visualization result not obvious.
